src/number_theory/modulus.rs

```diff
@@ -0,0 +1,47 @@
+///! Modulus Trait
+// Rust can perform the remainder operation, but not a modulus congruence operation
+use crate::number_theory::numbers::Integers;
+
+/// Implement the modulo operation
+///
+/// The modulus operator (%) is the remainder operation but not a congruency modulus
+/// See: https://en.wikipedia.org/wiki/Modular_arithmetic for more information
+///
+/// Two functions are implemented, one to find a possible congruency given the modulus, another to check if a and b are congruent given the modulus
+pub trait Modulus<T> {
+    fn modulus(self, b: T) -> T;
+    // fn is_congruent_modulus(self, b: T) -> bool;
```

Comment:
Two problems:
1. Not implemented or in TODO
2. Wrong number of arguments, a is congruent to b mod c